integration-tests/kafka/pom.xml

```diff
@@ -114,13 +135,31 @@
                         <artifactId>maven-failsafe-plugin</artifactId>
                         <executions>
                             <execution>
+                                <id>without-security</id>
+                                <goals>
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
+                                </goals>
+                                <configuration>
+                                    <systemProperties>
+                                        <native.image.path>
+                                            ${project.build.directory}/${project.build.finalName}-runner
+                                        </native.image.path>
+                                    </systemProperties>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <id>with-sasl-plaintext</id>
                                 <goals>
                                     <goal>integration-test</goal>
                                     <goal>verify</goal>
                                 </goals>
                                 <configuration>
                                     <systemProperties>
-                                        <native.image.path>${project.build.directory}/${project.build.finalName}-runner</native.image.path>
+                                        <kafka.security.protocol>SASL_PLAINTEXT</kafka.security.protocol>
```

Comment:
Why is this passed here and not in the Map produced by the Kafka Test Resource?

Reply:
This *instructs* the test resource to set up the broker to request authentication and to pass the required kafka-client properties on to quarkus.

Reply:
Implemented dedicated integration test, leaving the existing ones untouched.